src/App.jsx

```diff
@@ -1,11 +1,12 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom";
-import { Authentication, Home, WatchHistory, Playlist, LikedVideos, WatchLater, Profile} from "./pages";
+import { Authentication, Home, WatchHistory, Playlist, LikedVideos, WatchLater, Profile, Explore} from "./pages";
 import Mockman from "mockman-js";
 function App() {
   return (
     <Routes>
       <Route path="/" element={<Home />} />
+      <Route path="/explore" element={<Explore />} />
       <Route path="/history" element={<WatchHistory />} />
       <Route path="/playlist" element={<Playlist />} />
       <Route path="/liked" element={<LikedVideos />} />
```

Comment:
I would suggest you to give a try to [`useRoutes`](https://reactrouter.com/docs/en/v6/api#useroutes) hook, It's pretty straightforward.